pkg/email/trb_guidance_letter_internal_review_test.go

```diff
@@ -20,11 +20,11 @@ func (s *EmailTestSuite) TestTRBAdviceLetterInternalReviewEmail() {
 	getExpectedEmail := func(
 		leadName string,
 	) string {
-		trbAdminAdviceLetterLink := fmt.Sprintf(
+		trbAdminGuidanceLetterLink := fmt.Sprintf(
 			"%s://%s/%s",
 			s.config.URLScheme,
 			s.config.URLHost,
-			path.Join("trb", trbID.String(), "advice"),
+			path.Join("trb", trbID.String(), "guidance"),
```

Comment:
This URL is determined by the front end, so we'll just need to make sure the same change is made there.

Reply:
@aterstriep Just tagging you here for visibility on this.

Reply:
We can also always edit this later if we determine something else on the FE

Reply:
Thanks for pointing this out! I'll make sure it matches on the frontend.

Reply:
thanks Lee